openghg_defs/__init__.py

```diff
@@ -1,10 +1,16 @@
-from pathlib import Path
+import warnings as _warnings
 
-module_root = Path(__file__).resolve().parent
+from ._load import load_domain_info, load_internal_json, load_site_info, load_species_info
+from ._paths import get_domain_path, get_datapath
 
-data_path: Path = module_root / "data"
-
-site_info_file: Path = data_path / "site_info.json"
-species_info_file: Path = data_path / "species_info.json"
-domain_info_file: Path = data_path / "domain_info.json"
+_warnings.warn(
+    message="Direct access to filepaths will be removed in the next release."
+    + " Please start using either the loader functions or get_datapath.",
+    category=DeprecationWarning,
+)
+del _warnings
 
+data_path = get_datapath(filename="unused").parent
```

Comment:
I think this should either be removed or an empty file name passed here? I think including "unused" could just cause confusion.

Reply:
Yeah not sure why I did that, changed that.